hippo-digital/svp-tech-poc
# Task/view saved answers via nhs sub

**Note: This should be merged AFTER https://github.com/hippo-digital/svp-tech-poc/pull/2**

## Summary of changes

If the user logs in using an NHS account that has already submitted the form they are now forwarded to a 'review your submitted answers page'. They can view, but not edit, their responses on this page.

## Testing instructions

Run the following commands:

```bash
docker pull amazon/dynamodb-local
docker run -d -p 8000:8000 amazon/dynamodb-local
```

Then fire up the app as usual, log in using an NHS account and fill out the presented form. After hitting 'Submit' on the check your answers page the record will be pushed to the database and you will be routed back to the index page.

If you then log in again with the same account, rather than being shown the form, you will be redirected to a 'show saved answers page', which displays the summary list.
